.env.example

```diff
@@ -7,6 +7,10 @@ REACT_APP_HOME_NATIVE_NAME=POA
 REACT_APP_HOME_NETWORK_NAME="POA Sokol"
 REACT_APP_FOREIGN_NETWORK_NAME=Kovan
 
+# Uncomment the constant for the network that don't support events
+#REACT_APP_HOME_WITHOUT_EVENTS=1
+#REACT_APP_FOREIGN_WITHOUT_EVENTS=1
```

Comment:
Please update the README file with new parameters.

Reply:
Done